jina/peapods/runtimes/asyncio/rest/app.py

```diff
@@ -18,21 +18,6 @@
 from .....types.request import Request
 
 
-# TODO instead of this function make sure that only one 'oneof field' is used.
```

Comment:
Very cool that you found a way to get rid of this function call. It simplifies the implementation of the endpoint.